app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -35,7 +35,7 @@ EOF
 
 
 register_host() {
-  curl --cacert $SSL_CA_CERT --request POST <%= @registration_url %> \
+  curl -s --cacert $SSL_CA_CERT --request POST <%= @registration_url %> \
```

Comment:
Small nit, but in scripts it's better to use the long form. In the controller it's intended to be copied so there it's ok. This script it too long to copy. Then it becomes more important to understand it when you read it. With `curl --silent` you instantly understand what's going on while with `curl -s` it's much harder unless you use curl very often. That's why I'd use `--silent` in the templates and `-s` in the controller.